i18n/react-intl/en.json

```diff
@@ -17,7 +17,7 @@
   "overview": "Overview",
   "people": "People",
   "donate": "Donate",
-  "search": "Search",
+  "search": "Search with Google",
```

Comment:
Instead of changing the content, the search tag should be remove altogether and the corresponding changes should be made here: https://github.com/processing/processing-website/blob/649e59d9ddb7c9247e2884c1e6e55e136a20d78a/src/components/SearchBar.js#L43 
and the component should be cleaned up